extensions/typescript-language-features/src/features/task.ts

```diff
@@ -231,10 +232,13 @@ class TscTaskProvider implements vscode.TaskProvider {
 	}
 
 	private getLabelForTasks(project: TSConfig): string {
+		let projectPath: string | undefined = undefined;
 		if (project.workspaceFolder) {
-			return path.relative(project.workspaceFolder.uri.fsPath, project.path);
+			projectPath = path.relative(project.workspaceFolder.uri.fsPath, project.path);
+		} else {
+			projectPath = project.path;
 		}
-		return project.path;
+		return normalizePath(projectPath);
```

Comment:
@alexr00, done as suggested. Had to use a custom normalizePath() since path.posix.normalize() does not convert win32 paths to unix.

Reply:
One thing to note as you mentioned earlier, all the tasks.json files around the world which have windows path separators will show 'There are task errors' dialog with the below output:
![image](https://user-images.githubusercontent.com/9017268/57032706-a7418b80-6c68-11e9-8440-eeea0601bc64.png)

Reply:
That is correct. We did something similar with ${relativeFile} [here](https://github.com/Microsoft/vscode/issues/70050)

Reply:
@alexr00, If all is good, can you please approve this PR.

Reply:
@NizamLZ we are in our [endgame](https://github.com/Microsoft/vscode/issues/73175) now, which means that we are trying to only make critical changes. We were slowing down our changes last week too. Because of this, I'll need to wait until next week to merge this PR.

Reply:
Got it. I was just hoping for an approval of the PR, not merge. Non the less, I now understand that the PR is accepted. 

PS: This **'we are in the endgame now'** gives me chills! @Avengers